### ISSUE TITLE
Visualize Not-Active State in Elevation Graph

### ISSUE THREAD

**Issue by jifox** [nautobot/nautobot#2630]
### As ...

Ozzie - Operator

### I want ...

I want to see devices whit have `state != 'Active'` in elevation graphic. This helps me to decide where to mount new racked items.

### So that ...

when I look at the elevation graphic I can see which racked items are currently not active. (State not equal to "Active")

### I know this is done when...

- when I open an elevation view, I can see inactive racked items at a red unit height color.

![image](https://user-images.githubusercontent.com/8406301/196353154-25afdba4-bce3-4759-b48b-dc37e635353d.png)

### Optional - Feature groups this request pertains to.

- [ ] Automation
- [ ] Circuits
- [ ] DCIM
- [ ] IPAM
- [ ] Misc (including Data Sources)
- [ ] Organization
- [ ] Plugins (and other Extensibility)
- [ ] Security (Secrets, etc)
- [ ] Image Management
- [X] UI/UX
- [ ] Documentation
- [ ] Other (not directly a platform feature)

**Comment by glennmatthews** [comment]
Thanks for the request. A quick thought - we'd like to get *away* from having "magic" statuses (here, making `status.slug == "active"` have different behavior than any other arbitrarily named status). For example, maybe in a given environment your "active" status has been renamed to "production", the platform should be able to handle that gracefully.

So just thinking aloud here, maybe instead we could have *all* of the rack-unit numbers rendered similar to `label`s in the rest of the UI, color-coded to match the device's status color, **whatever** that may happen to be?

**Comment by jifox** [comment]
Yes, thats great. Use the status color to render the unit height numbers. So one can see even more information on a first sight

**Comment by itdependsnetworks** [comment]
Addressed in #6926
